packages/create-subscription/README.md

```diff
@@ -11,6 +11,16 @@ Other cases have **better long-term solutions**:
 * I/O subscriptions (e.g. notifications) that update infrequently should use [`simple-cache-provider`](https://github.com/facebook/react/blob/master/packages/simple-cache-provider/README.md) instead.
 * Complex libraries like Relay/Apollo should manage subscriptions manually with the same techniques which this library uses under the hood (as referenced [here](https://gist.github.com/bvaughn/d569177d70b50b58bff69c3c4a5353f3)) in a way that is most optimized for their library usage.
 
+## Limitations in async mode
+
+The main motivation for `create-subscription` is to provide a way for library authors to ensure compatibility with React's upcoming asynchronous rendering mode. `create-subscription` guarantees correctness in async mode, accounting for the subtle bugs and edge cases that a library author might otherwise miss.
+
+However, it acheives correctness by sometimes de-opting to synchronous mode, obviating async's benefits. This is an inherent limitation of storing state outside of React's managed state queue and rendering in response to a change event.
```

Comment:
achieves?

Reply:
"async's benefits" makes async a noun. It's a noun for us because we say it all the time but I'd say "the benefits of async rendering" here.

Reply:
Andrew is English much